docs/book/src/clusterctl/developers.md

```diff
@@ -98,6 +98,12 @@ please note that each `provider_repo` should have its own `clusterctl-settings.j
 
 The Docker provider is not designed for production use and is intended for development environments only.
 
+[kind] is not designed for production use.
```

Comment:
I would suggest dropping this statement because it is a duplication of the previous one